feature/google/src/testBc5SupportDev/java/BillingWrapperTest.kt

```diff
@@ -0,0 +1,2012 @@
+import android.app.Activity
```

Comment:
`git diff --no-index  feature/google/src/testBc5SupportDev/java/BillingWrapperTest.kt  feature/google/src/testLatestDependencies/java/BillingWrapperTest.kt`

Reply:
i have a follow-up ticket defined to split up the `BillingWrapperTest` files up more intelligently that will probably come next...